apps/web/src/app/api/commits/[commitUuid]/documents/route.ts

```diff
@@ -7,9 +7,9 @@ export async function GET(
 ) {
   try {
     const staged = Boolean(req.nextUrl.searchParams.get('staged') || false)
-    const nodes = await materializeDocumentsAtCommit({ commitUuid, staged })
+    const documents = await materializeDocumentsAtCommit({ commitUuid, staged })
 
-    return NextResponse.json(nodes)
+    return NextResponse.json(documents)
```

Comment:
Proposal. Instead of returning the documents here I would return the tree and call this API something like 
`/api/sidebar/commitUuid` 

Let's data over the wire